TenureInformationApi.Tests/V1/Helper/DateHandlingHelper.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using AutoFixture.Kernel;
+
+namespace TenureInformationApi.Tests.V1.Helper
+{
+    public class UtcDateTimeHelper : ISpecimenBuilder
+    {
+        public object Create(object request, ISpecimenContext context)
+        {
+            var type = request as Type;
+            if (type == typeof(DateTime))
+            {
+                return DateTime.UtcNow;
+            }
+            return new NoSpecimen();
+        }
+    }
```

Comment:
Not sure about this approach...

If the intention is to hardcode any date being created to `DateTime.UtcNow`, then won't the
```cs
_fixture
     .Build<MyClassContainingDateTime>()
     .With(c => c.MyDateField, DateTime.UtcNow)
     .Create();
```
do the job ?

If the intention is to merely create DateTime that's a 'now' date, then the `DateTime.UtcNow` without any data faking package would be better.